.github/workflows/CI.yaml

```diff
@@ -39,23 +44,19 @@ jobs:
         df -h
         ulimit -a
 
-
     # More info on options: https://github.com/conda-incubator/setup-miniconda
     # An environment for the minimum versions 
     - uses: conda-incubator/setup-miniconda@v2
       with:
-        python-version: ${{ matrix.python-version-min }}
-        environment-file: devtools/conda-envs/test_min.yaml
+        python-version: ${{ matrix.python-version }}
+        environment-file: devtools/conda-envs/test_${{ matrix.env_file }}.yaml
```

Comment:
we use the `env_file` variable to selectively choose which environment file is passed through.